deepchem/models/tensorgraph/models/test_sascore.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+import deepchem
+import numpy as np
+from deepchem.models import TensorGraph
+
+
+class TestSaScoreModel(unittest.TestCase):
+
+  def test_save_load(self):
+    """Test creating an Estimator from a ScScoreModel."""
```

Comment:
The documentation should be a test for saving and loading the model. This applies for the estimator test.